README.md

```
@@ -3,564 +3,152 @@
-Server: Backend Key Data + Ready for Query
-    ↓
-Connection Ready
+```bash
+bunx eas build --platform ios
+bunx eas build --platform android
 ```
 
-### Query Execution
+## Architecture
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)